charts/postgresql/values.yaml

```diff
@@ -0,0 +1,14 @@
+architecture: replication
+global:
+  storageClass: local-storage
+auth:
+  replicationPassword: externalPostgresql
+  postgresPassword: externalPostgresql
+  username: externalPostgresql
+  password: externalPostgresql
+  database: externalPostgresql
+readReplicas:
+  name: read
+  replicaCount: 2
+replication:
+  numSynchronousReplicas: 2
```

Comment:
why do you need more than one replica in your use case (bots)? Would it not be sufficient to have one single replica (and no read replicas)?

Perhaps the first line `architecture: replication` could be changed to `architecture: standalone` (already the default). If not, please motivate why you need higher availability here at this level, and what this chart is used for (and also where it is used).